pkg/proxy/util/conntrack.go

```diff
@@ -18,30 +18,27 @@ package util
 
 import (
 	"fmt"
+	"strconv"
 	"strings"
 
 	"k8s.io/utils/exec"
-
-	"github.com/golang/glog"
 )
 
 // Utilities for dealing with conntrack
 
 const noConnectionToDelete = "0 flow entries have been deleted"
 
-// DeleteServiceConnection uses the conntrack tool to delete the conntrack entries
-// for the UDP connections specified by the given service IPs
-func DeleteServiceConnections(execer exec.Interface, svcIPs []string) {
-	for _, ip := range svcIPs {
-		glog.V(2).Infof("Deleting connection tracking state for service IP %s", ip)
-		err := ExecConntrackTool(execer, "-D", "--orig-dst", ip, "-p", "udp")
-		if err != nil && !strings.Contains(err.Error(), noConnectionToDelete) {
-			// TODO: Better handling for deletion failure. When failure occur, stale udp connection may not get flushed.
-			// These stale udp connection will keep black hole traffic. Making this a best effort operation for now, since it
-			// is expensive to baby-sit all udp connections to kubernetes services.
-			glog.Errorf("conntrack returned error: %v", err)
-		}
+// DeleteServiceConnections uses the conntrack tool to delete the conntrack entries
+// for the UDP connections specified by the given service IP
+func DeleteServiceConnections(execer exec.Interface, ip string) error {
```

Comment:
Glad to receive your feedback. @freehan 

 I agree this function needs a better name. Since the parameter of this function is not limited to Cluster IP, so can we rename it to `ClearUDPConntrackForIP`? 

Please let me know if you have better ideas :) Thanks!